cvat/schema.yml

```diff
@@ -9274,6 +9811,115 @@ components:
       - password1
       - password2
       - username
+    Request:
+      type: object
+      properties:
+        status:
+          allOf:
+          - $ref: '#/components/schemas/RequestStatusEnum'
+          readOnly: true
+        message:
+          type: string
+          readOnly: true
+        id:
+          type: string
+          readOnly: true
+        operation:
+          allOf:
+          - $ref: '#/components/schemas/RequestDataOperation'
+          readOnly: true
+        progress:
+          type: number
+          format: double
+          maximum: 1
+          minimum: 0
+          readOnly: true
+          nullable: true
+        created_date:
+          type: string
+          format: date-time
+          readOnly: true
+        enqueued_date:
+          type: string
+          format: date-time
+          readOnly: true
+          nullable: true
```

Comment:
Is there a meaningful difference between `created_date` and `enqueued_date`, as far as the user is concerned? If a request was created, it must have also been enqueued.

Reply:
Probably not for now, but I think it depends on whether we're going to inform the user about deferred jobs or continue to mask them as queued.

Reply:
Then let's only expose one of these fields until such a time when there's a reason to expose the other one.

(FWIW, my vote is for `created_date`, as that would be consistent with other API resources.)